srml/grandpa/src/lib.rs

```diff
@@ -151,6 +152,12 @@ decl_storage! {
 
 		/// `true` if we are currently stalled.
 		Stalled get(stalled): Option<(T::BlockNumber, T::BlockNumber)>;
+
+		/// The current set-id.
+		CurrentSetId get(current_set_id) build(|_| fg_primitives::genesis_set_id()): SetId;
```

Comment:
Why placing a function which returns 0? Won't it always be zero? Under what conditions would this not be initialised to zero?

Reply:
It's always going to be 0 for the moment (although later on, chains might base it off of the genesis hash / fork ID). Regardless, I'd prefer not to scatter the knowledge and justification that it's 0 too far and wide.